### PR TITLE
[1LP][RFR] Fixed navigation for Node Alert Profile

### PR DESCRIPTION
**Pull request by quarckster**
{{pytest: -v -k test_alert_profile_crud}}

### CODE REVIEW

File: cfme/control/explorer/alert_profiles.py [hunk 1]
```diff
@@ -308,4 +308,9 @@ class VMInstanceAlertProfile(BaseAlertProfile):
 
 class NodeAlertProfile(BaseAlertProfile):
 
-    TYPE = "Node"
+    @property
+    def TYPE(self):  # noqa
+        if self.appliance.version < "5.9.2":
```
Comment (CharlleDaniel) [inline cfme/control/explorer/alert_profiles.py]: `<` doesn't compares strings as expected in this case. Try "5.10.2". It will return True.

Comment (mshriver) [reply cfme/control/explorer/alert_profiles.py]: @CharlleDaniel This appears to be behaving correctly to me:

```
In [5]: node
Out[5]: NodeAlertProfile(description='test-profile', alerts=None, notes=None)

In [6]: node.TYPE
Out[6]: 'Container Node'

In [7]: app.version
Out[7]: Version('5.9.2.0')
```

```
In [5]: node = profiles.instantiate(NodeAlertProfile, description='test-profile')

In [6]: node.TYPE
Out[6]: 'Node'

In [7]: app.version
Out[7]: Version('5.9.1.1')
```

Comment (quarckster) [reply cfme/control/explorer/alert_profiles.py]: `appliance.version` refers to `miq_version.Version`. Feel free to study it.

Comment (CharlleDaniel) [reply cfme/control/explorer/alert_profiles.py]: @quarckster  We still have a string comparison. Here is a running example:
image

Feel free to run [https://trinket.io/python/6edc87ab97](https://trinket.io/python/6edc87ab97)

Comment (quarckster) [reply cfme/control/explorer/alert_profiles.py]: @CharlleDaniel your example shows strings comparison, but in my PR I compare string and instance of `miq_version.Version` class https://github.com/RedHatQE/miq_version/blob/master/miq_version/__init__.py. It has implemented `__eq__`, `__lt__` and `__cmp__`. Please study the framework and its classes.

Comment (CharlleDaniel) [reply cfme/control/explorer/alert_profiles.py]: @quarckster I didn't have noticed that, thank you.

**Pull request merged**